frontend/src/tests/ForestClientInput.spec.ts

```diff
@@ -102,4 +121,286 @@ describe('ForestClientInput', () => {
+
+        const validationError = wrapper.findAll('#forestClientInputValidationError');
+        // we need to have two error messages as well (inactive and invalid)
+        expect(validationError).toHaveLength(2);
+        // The first entry in the forest client card should be equal to the first active input
+        expect(
+            setVerifiedForestClients
+                ? setVerifiedForestClients[0][0]
+                : undefined
+        ).toEqual(TEST_SUCCESS_FOREST_CLIENT_NUMBER);
+        // The second entry in the forest client card should be equal to the second active input
+        expect(
+            setVerifiedForestClients
+                ? setVerifiedForestClients[1][0]
+                : undefined
+        ).toEqual(TEST_SUCCESS_FOREST_CLIENT_NUMBER_2);
+        // the forest client information card shows and verify the information
+        expect(wrapper.html().includes('Verified Client ID information')).toBe(
+            true
+        );
+        expect(
+            wrapper
+                .find('.client-id-wrapper')
+                .text()
+                .includes(TEST_SUCCESS_FOREST_CLIENT_NUMBER)
+        ).toBe(true);
+        expect(
+            wrapper
+                .find('.org-name-wrapper')
+                .text()
+                .includes(TEST_FOREST_CLIENT_NAME)
+        ).toBe(true);
+        expect(
+            wrapper.find('.org-status-wrapper').text().includes('Active')
+        ).toBe(true);
+        // the forest client input should still display the error input numbers
+        expect(inputField.value).toContain(TEST_INACTIVE_FOREST_CLIENT_NUMBER);
+        expect(inputField.value).toContain(TEST_INVALID_FOREST_CLIENT_NUMBER);
+    });
+
+    //TODO - The test case below needs to adjust the mockImplementation for the api call to have a non blocking delay
+    // so the loadingState and the button label could be tested. I have tried some approaches but none seem to work:
+    // - Use of setTimeout doesn't work well in testing (the timout is not applied)
+    // - Use of vi.useFakeTimers() does apply delay, but I am not sure if it is a blocking or non blocking task
+
+    // it('should test the adding button in on loading while we call the api', async () => {
+    //     const input = wrapper.find('#forestClientInput');
+    //     const inputField: HTMLInputElement = input.element as HTMLInputElement;
+    //     await input.setValue(TEST_SUCCESS_FOREST_CLIENT_NUMBER);
+    //     expect(inputField.value).toBe(TEST_SUCCESS_FOREST_CLIENT_NUMBER);
+    //     expect(isLoading()).toBe(false);
+
+    //     await wrapper.find("[aria-label='Add Client Numbers']").trigger('click');
+    //     await flushPromises();
+
+    //     expect(isLoading()).toBe(true);
+    //     expect(wrapper.find("[aria-label='Add Client Numbers']").element.textContent).contains('Loading')
+    // });
+
+    it('should test the delete forest client number case', async () => {
```

Comment:
Hi Nick, when we test the delete functionality, there is no need to verify the setVerifiedForestClients is called, because we already tested it multiple times in the previous test cases, and there is no need to verify the card content. 

I feel it's better to just add one forest client number, and verify the card is displayed and has one row, and when you click delete, the card is removed.